yaml/stage.go

```diff
@@ -120,3 +122,32 @@ func (s StageSlice) MarshalYAML() (interface{}, error) {
 
 	return output, nil
 }
+
+// MergeEnv takes a list of environment variables and attempts
+// to set them in the stage environment. If the environment
+// variable already exists in the stage, than this will
+// overwrite the existing environment variable.
+func (s *Stage) MergeEnv(environment map[string]string) error {
+	// check if the step container is empty
```

Comment:
I think the comment here has a typo i.e. ~`container`~

```suggestion
	// check if the stage environment is empty
```